### PR TITLE
Use "channel" as the "message_thread_id" when using the Telegram template

### PR DESCRIPTION
**Pull request by yegle**
See reference in https://core.telegram.org/bots/api#sendmessage

### CODE REVIEW

File: pkg/unpackerr/webhooks_templates.go [hunk 1]
```diff
@@ -77,6 +77,7 @@ const WebhookTemplateNotifiarr = `{
 
 const WebhookTemplateTelegram = `{
   "chat_id": "{{nickname}}",
+  {{- if channel }}"message_thread_id": {{channel}}{{end -}},
```
Comment (davidnewhall) [inline pkg/unpackerr/webhooks_templates.go:80]: The comma at the end should probably be inside the if/end blocks.

**Review by davidnewhall — CHANGES_REQUESTED** [review]
Can you tell us what this does? What problem it solves? Why someone would want to provide the new field to telegram?

**Comment by yegle** [comment]
This was a new feature from telegram, where you can specify a `message_thread_id` and it will become part of a named topic in the chat room.

I guess maybe it would be easier to override `webhook.template_path` and add the `messate_thread_id`.

**Comment by davidnewhall** [comment]
I don't mind adding it. Just wanted to know more about it since I don't send telegram notifications myself. I also wanted to make sure that comma wasn't out of place. You're welcome to re-open this and fix the comma. Thank you!

**Pull request closed without merging**